docs/installation-configuration.md

```diff
@@ -45,12 +45,12 @@ Example for using the `runtimeArtifact` parameter:
 </plugin>
 ```
 
-The coordinates for `runtimeArtifact` can be overridden using `libertyRuntimeGroupId`, `libertyRuntimeArtifactId`, and `libertyRuntimeVersion`. These can be set using command line properties, pom.xml properties, or additional plugin configuration. Empty or `null` values will result in a default value overriding the respective `runtimeArtifact` coordinate value. More information on these properties can be found in [common parameters](docs/common-parameters.md#common-parameters).
+The coordinates for `runtimeArtifact` can be overridden using `libertyRuntimeGroupId`, `libertyRuntimeArtifactId`, and `libertyRuntimeVersion`. These can be set using command line properties, pom.xml properties, or additional plugin configuration. Empty or `null` values will result in a default value overriding the respective `runtimeArtifact` coordinate value. More information on these properties can be found in [common parameters](common-parameters.md#common-parameters).
 
 Example of overriding the `runtimeArtifact` parameter through the command line:
 
 ```
-mvn clean install -Dliberty.runtime.groupId=io.openliberty -Dliberty.runtime.artifactId=openliberty-runtime -Dliberty.runtime.version=21.0.0.9
+mvn clean install -DlibertyRuntimeGroupId=io.openliberty -DlibertyRuntimeArtifactId=openliberty-runtime -DlibertyRuntimeVersion=21.0.0.9
```

Comment:
Why did you change these? The previous values work. There are test cases that use them. Did you add tests that show these new values work? If both work, then we should document both.

Reply:
No actually this does not work... I had changed it, tested it, found it didn't work, but somehow missed pushing it back.   You can use one or the other in parameter config but for a property you can only use what we had.  Confusing... Thanks for catching !